website/content/en/docs/overview/operator-capabilities.md

```diff
@@ -8,7 +8,7 @@ Operators come in different maturity levels in regards to their lifecycle manage
 
 ![operator-capability-level](/operator-capability-level.png)
 
-Each capability level is associated with a certain set of management features the Operator offers around the managed workload. Operator that do not manage a workload and/or are delegating to off-clusters orchestration services would remain at Level 1. Capability levels are accumulating, i.e. Level 3 capabilities require all capabilities desired from Level 1 and 2.
+Each capability level is associated with a certain set of management features the Operator offers around the managed workload. Operator that do not manage a workload and/or are delegating to off-clusters orchestration services would remain at Level 1. Capability levels range from level 1 to level 5. Each capability represents its own set of features and may be independent of each other.
```

Comment:
I think the word "range" might still communicate a hierarchy of functionality. Maybe we can try something like "Capability levels are labeled from level 1 to level 5?"

Reply:
was going to say "designated"

Reply:
That's a great word, "Designated" is better

Reply:
Done. Let me know if that's better.